.env.test

```diff
@@ -2,16 +2,18 @@
 DATABASE_NAME=postgres
 DATABASE_USER=postgres
 DATABASE_PASSWORD=root
+POSTGRES_PASSWORD=root
 DATABASE_HOST=localhost
-DATABASE_PORT=5430
+DATABASE_PORT=5432
```

Comment:
I'm personally a fan of keeping the database port at 5430. Otherwise, you can't execute the tests while running the application. That's annoying sometimes 🙃 What do you think about it?

Reply:
since it is now the same database, the ports will be overriden, so when running tests, the dev environment will stop and vice verca

Reply:
It make no difference in swithcing the port, because the other instance would not be running anymore

Reply:
Yes sure :)

Reply:
Before we had two different postgresql containers (dev and test), now it is one for both